packages/sdk-react/test/useGraph.test.ts

```diff
@@ -1,20 +1,25 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import "mocha";
-import { assert, expect } from "chai";
+/**
+ * @jest-environment jsdom
+ */
+
 import { renderHook } from "@testing-library/react-hooks";
 import { Client, GraphError } from "@microsoft/microsoft-graph-client";
 import { useGraph } from "../src/useGraph";
 import { TeamsFx, ErrorWithCode, ErrorCode } from "@microsoft/teamsfx";
 import { act } from "react-test-renderer";
-import * as sinon from "sinon";
 
-const sandbox = sinon.createSandbox();
+require("isomorphic-fetch");
 
 describe("useGraph() hook tests", () => {
-  afterEach(() => {
-    sandbox.restore();
+  let spyTeamsFxLogin: jest.SpyInstance;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
It's better to put restore steps in afterEach()

Reply:
Thanks for the comments. The restore steps in the tests are all moved to afterEach() now.